src/api/services/usersService.ts

```diff
@@ -76,25 +78,15 @@ export const enrichWithUserFromParams = async (
   next: NextFunction
 ) => {
   const { userId } = userIdParser.parse(req.params);
-  const traceId = getTraceId(req);
   const user = await User.query().findById(userId);
   if (!user) {
     return res.status(404).send([{ message: 'The user cannot be found.' }]);
   }
-  if (
```

Comment:
The above changes mean that the user must already be related to the participant by now

Reply:
Sent you a DM.